tests/js/jest.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+	preset: '@wordpress/jest-preset-default',
+	rootDir: '../../',
+	verbose: true,
+	moduleDirectories: [ 'node_modules', '<rootDir>/client' ],
+	restoreMocks: true,
+	transform: {
+		'^.+\\.jsx?$': 'babel-jest',
+	},
+	transformIgnorePatterns: [ 'node_modules/(?!(@woocommerce/.+)/)' ],
+	moduleNameMapper: {
+		'\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/tests/js/jest-file-mock.js',
+		'^react$': '<rootDir>/node_modules/react',
+		'^react-dom$': '<rootDir>/node_modules/react-dom',
+		'^wcstripe(.*)$': '<rootDir>/client$1',
```

Comment:
Added an alias for WP so that we can include with `import * from 'wcstripe/*` in our JS files